### PR TITLE
Project - Members Only!

### PR DESCRIPTION
**Pull request by jrai0792**
This project deals with the authentication and authorization of users.
I have used the 'devise' gem for the authentication and made the model user using the gem.
Created the controller for new, create and index methods and corresponding views.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,39 @@
...
+To get this project set up on your local machine, follow these simple steps:
 
-* Database initialization
+- Open Terminal.
+- Navigate to your desired location to download the contents of this repository.
+- Copy and paste the following code into the Terminal: git clone https://github.com/jrai0792/members-only
+- Run cd members-only.
+- Run bundle install to get the necesary gems.
+- Run rails db:create.
+- Run rails db:migrate.
```
Comment (lekegitrepo) [inline README.md:25]: - **_Add instructions on how to run the project locally_**

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  # GET /posts/new
+  def new
+    @post = Post.new
+  end
+
+  # POST /posts
+  # POST /posts.json
+  def create
+    @post = Post.new(post_params)
+    @post.user_id = current_user.id
```
Comment (lekegitrepo) [inline app/controllers/posts_controller.rb:31]: - **_(optional) You can also put it in one line: `@post = current_user.posts.build(post_params)`_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
#### Hi @jrai0792 :clap: :clap:
#### Well done :muscle: the project is well organized and easy to understand. However, a few issues need to be fixed before this project can be approved. Check my comments below. The comments tagged (optional) are not mandatory.

**Review by Dandush03 — APPROVED** [review]
# Status: Project Approved :rocket: 
# Great work implementing changes
# Happy Coding :nerd_face:

**Pull request merged**